src/lib/db/group-store.ts

```diff
@@ -18,12 +19,23 @@ const rowToGroup = (row) => {
         id: row.id,
         name: row.name,
         description: row.description,
-        createdAt: row.createdAt,
-        createdBy: row.createdBy,
+        createdAt: row.created_at,
+        createdBy: row.created_by,
     });
 };
 
-const groupToRow = (user: ICreateGroup) => ({
+const rowToGroupUser = (row) => {
+    if (!row) {
+        throw new NotFoundError('No group user found');
```

Comment:
What happens if you're trying to query a group that has no users yet?

Reply:
Actually by design, this is not allowed. There must be always a owner user, otherwise it does not allow creating group.

Reply:
Excellent point! Good call!